berkeley-function-call-leaderboard/bfcl/model_handler/hammer_handler.py

```diff
@@ -0,0 +1,173 @@
+import json
+
+from bfcl.model_handler.oss_handler import OSSHandler
+from bfcl.model_handler.model_style import ModelStyle
+
+
+
+TASK_INSTRUCTION = """You are a tool calling assistant. In order to complete the user's request, you need to select one or more appropriate tools from the following tools and fill in the correct values for the tool parameters. Your specific tasks are:
+1. Make one or more function/tool calls to meet the request based on the question.
+2. If none of the function can be used, point it out and refuse to answer.
+3. If the given question lacks the parameters required by the function, also point it out.
+"""
+
+FORMAT_INSTRUCTION = """
+The output MUST strictly adhere to the following JSON format, and NO other text MUST be included.
+The example format is as follows. Please make sure the parameter type is correct. If no function call is needed, please directly output an empty list '[]'
+```
+[
+    {"name": "func_name1", "arguments": {"argument1": "value1", "argument2": "value2"}},
+    ... (more tool calls as required)
+]
+```
+"""
+
+model_name1  = ''
+
+class hammerHandler(OSSHandler):
+    def __init__(
+        self, model_name, temperature=0.001, top_p=1, max_tokens=512, dtype="bfloat16"
+    ) -> None:
+        super().__init__(model_name, temperature, top_p, max_tokens, dtype)
+        self.model_style = ModelStyle.OSSMODEL
+        global model_name1
```

Comment:
What's the purpose of this `model_name1`?

Reply:
Sorry, model_name1 is redundant and I forgot to delete it. Can you help me delete it, or do I need to delete it and resubmit the pull request

Reply:
Hi, I have already fixed it. Thanks for the reminder.